# Generated for NaoQi method ALSpeechRecognition.setWordListAsVocabulary
# Sets the list of words (vocabulary) that should be recognized by the speech recognition engine.

# List of words that should be recognized
string[] vocabulary
---


